### PR TITLE
HERITAGE-65 Add related tags to details page

### PR DESCRIPTION
**Pull request by garethmead**
Ticket URL: [HERITAGE-65](https://national-archives.atlassian.net/browse/HERITAGE-65)

## About these changes

Add related tags section to details page

## How to check these changes

Visit details page from search results. Eg - `/catalogue/id/C8077549/` and observe tags in right hand column

The table is currently duplicated to show how the mark up should be set with or without the tags as we don't yet have the logic / data for this

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [ ] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

[HERITAGE-65]: https://national-archives.atlassian.net/browse/HERITAGE-65?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/includes/related-tags.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                </a>
+            </span>
+        </dd>
+
+        <dt>Other:</dt>
+        <dd>
+            <span class="ohos-tag ohos-tag--other">
+                <span class="ohos-tag__inner">Other</span>
+
+                <a href="https://test-deploy-ohos-xjy4lfq-rasrzs7pi6sd4.uk-1.platformsh.site/search/catalogue/?filter_keyword=&covering_date_from_0=&covering_date_from_1=&covering_date_from_2=&covering_date_to_0=&covering_date_to_1=&covering_date_to_2=&collection=ADM&opening_start_date_0=&opening_start_date_1=&opening_start_date_2=&opening_end_date_0=&opening_end_date_1=&opening_end_date_2=&q=&sort_by=&sort_order=asc&group=tna" class="ohos-tag__link">
```
Comment (ahosgood) [inline templates/includes/related-tags.html:63]: Do these need a `title` attribute? I'm just worried that the link just reads "Collection"

**Review by ahosgood — APPROVED** [review]
Approved once the comments are addressed

File: templates/includes/related-tags.html (hunk 1 above)
Comment (garethmead) [reply templates/includes/related-tags.html]: Updated with a title attribute "See other (tag name) results in the collection". Once we have the data, tag name will be replaced with the actual name

**Review by TNA-Allan — APPROVED** [review]
Tested, looks good. Thanks @garethmead

**Pull request merged**